examples/ansible/setup_neuvector.yml

```diff
@@ -0,0 +1,113 @@
+  hosts: alphost
+  vars:
+    neuvector_state: present
+    workload:
+      name: neuvector
+      image: registry.opensuse.org/suse/alp/workloads/bci_containerfiles/suse/alp/workloads/neuvector-demo:latest
+
+  tasks:
+    - name: Setup NeuVector
+      when:
+        - neuvector_state == "present"
+      block:
+        - name: Check if workload {{ workload.name }} is already installed
+          become: true
+          containers.podman.podman_container_info:
+            name: "{{ workload.name }}-allinone"
+          register: workload_info
+          changed_when: false
+
+        - name: Set SELinux into permissive mode
+          become: true
+          ansible.posix.selinux:
+            policy: targeted
+            state: permissive
+          when:
+            - workload_info.containers | length == 0
+
+        - name: Retrieve image for workload {{ workload.name }}
+          become: true
+          containers.podman.podman_image:
+            name: "{{ workload.image }}"
+            state: present
+          register: retrieve_workload_image
+          when:
+            - workload_info.containers | length == 0
+
+        - name: Execute nevector runlabel INSTALL
+          become: true
+          ansible.builtin.command: >-
+            podman container runlabel install {{ workload.image }}
+          register: workload_runlabel_install
+          changed_when:
+            - ('already exist' not in workload_runlabel_install.stdout)
+            - workload_info.containers | length == 0
+
+        - name: Start NeuVector service
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            state: started
+          register: workload_service_start
+
+        - name: enable service to start on a reboot
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            enabled: yes
+          when:
+            - workload_service_start is success
+
+        - name: Print message connect to NeuVector
+          ansible.builtin.debug:
+            msg: >-
+              NeuVector is running on https://{{ ansible_default_ipv4.address }}:8443
+              You need to accept the warning about the self-signed SSL certificate
+              and log in with the following default credentials: admin / admin.
+          when:
+            - workload_service_start is success
+
+    - name: Remove NeuVector
+      when:
+        - neuvector_state == "absent"
+      block:
+        - name: Stop NeuVector service
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            state: stopped
+            enabled: no
+          ignore_errors: yes
+
+        - name: Execute nevector runlabel UNINSTALL
+          become: true
+          ansible.builtin.command: >-
+            podman container runlabel uninstall {{ workload.image }}
+          register: workload_runlabel_uninstall
```

Comment:
What is this needed for?

Reply:
You register Ansible variable `workload_runlabel_uninstall` but you don't use it.